kubetools_client/config.py

```diff
@@ -65,29 +65,23 @@ def load_kubetools_config(
     else:
         # Look in this directory
```

Comment:
This comment is now obsolete

Reply:
https://github.com/EDITD/kubetools-client/pull/13/commits/dbc73ee5efef9eba7a9a41b4206a18b828395dc7